warehouse/redshift.go

```diff
@@ -191,14 +207,15 @@ func (rs *Redshift) EnsureCorrectExportTable() error {
 	return nil
 }
 
-func (rs *Redshift) CopyInData(s3file string, headers []string) error {
-	log.Printf("Loading in data from %s", s3file)
-	copy := fmt.Sprintf("COPY %s (%s) FROM '%s' CREDENTIALS '%s' DELIMITER ',' REGION '%s' FORMAT AS CSV ACCEPTINVCHARS;",
-		rs.conf.Redshift.ExportTable, strings.Join(headers, ","), s3file, rs.conf.Redshift.Credentials, rs.conf.S3.Region)
+// CopyInData copies data from the given s3File to the export table
+func (rs *Redshift) CopyInData(s3file string) error {
```

Comment:
I removed the headers from the the copy command since we are now guaranteed matching column order between the csv and the export table